src/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -43,7 +46,13 @@ export function QueryParameterActionsProvider() {
     )
   );
 
+  const addTermsFilter = (state) => (field, values, operation) => {
+    const { queryParameters: { indexPatternId } } = state;
+    filterManager.add(field, values, operation, indexPatternId);
```

Comment:
Should we also call `$scope.indexPattern.popularizeField(field, 1);` like in [discover](https://github.com/elastic/kibana/blob/9c37e6c4c1c89a0c311faf8dbbaad008ac25c612/src/core_plugins/kibana/public/discover/controllers/discover.js#L487-L487)?